media-sound/denemo/metadata.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+<maintainer type="person">
+	<email>waebbl@gmail.com</email>
+	<name>Bernd Waibel</name>
+</maintainer>
+<!-- TODO: Talk to sound project or remove it
+<maintainer type="project">
+	<email>sound@gentoo.org</email>
+	<name>Gentoo Sound project</name>
+</maintainer>
+-->
+<maintainer type="project">
+	<email>proxy-maint@gentoo.org</email>
+	<name>Proxy Maintainers</name>
+</maintainer>
+<longdescription>
+	Denemo is a music notation editor which uses
+	<pkg>media-sound/lilypond</pkg> to typeset music notation with LaTeX.
+</longdescription>
+<use>
+	<flag name="alsa">Enable alsa support through <pkg>media-libs/alsa-lib</pkg></flag>
+	<flag name="aubio" restrict="&gt;media-sound/denemo-2.2.0">
+		Enable <pkg>media-libs/aubio</pkg> support</flag>
+	<flag name="coverage" restrict="&gt;media-sound/denemo-2.2.0">
+		Enable coverage testing of the code</flag>
+	<flag name="evince" restrict="&gt;media-sound/denemo-2.2.0">
+		Use <pkg>app-text/evince</pkg> document viewer</flag>
+	<flag name="doc" restrict="&gt;media-sound/denemo-2.2.0">
+		Use <pkg>dev-util/gtk-doc</pkg> to build documentation in html format</flag>
+	<flag name="gtk3" restrict="&gt;media-sound/denemo-2.2.0">
+		Enable support for the Gimp Toolkit. If enabled uses slot 3 of
```

Comment:
Don't use the 'Gimp Toolkit' name, it's confusing these days. Everybody calls it just 'GTK+'.

Reply:
changed.